pipelines/validation/h1c_idr3_2/h1c_idr3_v2_validation.toml

```diff
@@ -2,12 +2,22 @@
 makeflow_type = "analysis"
 path_to_do_scripts = "/lustre/aoc/projects/hera/Validation/test-4.1.0/software/hera_pipelines/pipelines/validation/h1c_idr3_2/task_scripts"
 path_to_a_priori_flags = "/lustre/aoc/projects/hera/H1C_IDR3/IDR3_2/h1c_idr3_software/hera_pipelines/pipelines/h1c/idr3/v2/a_priori_flags"
+path_to_sim_files = "/lustre/aoc/projects/hera/Validation/H1C_IDR3/chunked_data"
```

Comment:
You don't need these two here.

Reply:
Oops, guess I messed up handling the merge conflicts... those were moved in the latest version.